README.md

```diff
@@ -69,13 +69,19 @@ Please contact mehdi@umich.edu if you have any questions.
 
 #Generators
 
--**[temp-sense-gen:](https://github.com/idea-fasoc/OpenFASOC/tree/main/openfasoc/generators/temp-sense-gen)** 
--**[cryo-gen:](https://github.com/idea-fasoc/OpenFASOC/tree/main/openfasoc/generators/cryo-gen)**
--**[dcdc-gen:](https://github.com/idea-fasoc/OpenFASOC/tree/main/openfasoc/generators/dcdc-gen)**
--**[gdsfactory:](https://github.com/idea-fasoc/OpenFASOC/tree/main/openfasoc/generators/gdsfactory)**
--**[lc-dco:](https://github.com/idea-fasoc/OpenFASOC/tree/main/openfasoc/generators/lc-dco)**
--**[ldo-gen:](https://github.com/idea-fasoc/OpenFASOC/tree/main/openfasoc/generators/ldo-gen)**
--**[scpa-gen:](https://github.com/idea-fasoc/OpenFASOC/tree/main/openfasoc/generators/scpa-gen)**
+- **[temp-sense-gen:](https://github.com/idea-fasoc/OpenFASOC/tree/main/openfasoc/generators/temp-sense-gen)** 
```

Comment:
Do I commit this changes to pull request #51 or #52 to avoid merge conflict

Reply:
commit all your changes to #51 and close that pull request.. @irakobi, If @kalabYibeltal is done with all his changes, you first pull changes to your clone and then continue your work. But I think you will end up with conflicts anyways because you both are working on same lines TBH.

Reply:
I am not done with #51 PR, The task to add readme pages to the generators still reamains.

Reply:
In order to avoid conflict as we were working on the same lines, I suggest that @kalabYibeltal can finish all changes requested on the main page(README) including the change I have made which are right. Then I will cancel my PR #52 then pull the change. After all I will open another PR on Adding README for each generator from their directories as requested by  Mehdi.

Reply:
As you wish!